app/BigQuery_service.py

```diff
@@ -0,0 +1,41 @@
+from google.cloud import bigquery
+from google.oauth2 import service_account
+import pandas as pd
+
+credentials = service_account.Credentials.from_service_account_file(
```

Comment:
let's delete this unused creds variable in favor of just setting the client within the class' init method (like you're doing already)